RELEASE/scripts/autoscend/auto_restore.ash

```diff
@@ -1088,7 +1088,7 @@ boolean __restore(string resource_type, int goal, int meat_reserve, boolean useF
 
   skill pick_blood_skill(int final_hp){
     boolean bloodBondAvailable = auto_have_skill($skill[Blood Bond]) &&
-      auto_have_familiar($familiar[Mosquito]) && //checks if player can use familiars in this run
+      canChangeToFamiliar($familiar[Mosquito]) &&
```

Comment:
```suggestion
      pathAllowsFamiliar() &&
```
Previous change fixed it for pokefam, but not for 100% familiar runs.
This fixes it for both. However, it should not be committed until PR#324 has been committed first